backend/app/services/collections/helpers.py

```
@@ -56,24 +56,51 @@ def extract_error_message(err: Exception) -> str:
 
 
 def batch_documents(
-    document_crud: DocumentCrud, documents: List[UUID], batch_size: int
-):
-    """Batch document IDs into chunks of size `batch_size`, load each via `DocumentCrud.read_each`,
-    and return a list of document batches."""
+    document_crud: DocumentCrud, documents: List[UUID]
+) -> List[List[Document]]:
```

Ruff (0.15.6)

[error] Undefined name `Document`

(F821)